### DISCUSSION TITLE
How to define a panoramic fisheye camera?

### DISCUSSION THREAD

**Discussion by TaoHuUMD** [facebookresearch/pytorch3d#539] [Q&A]
Hi, 

Any suggestions to add a panoramic fisheye camera model ? 

We still inherit from the CamerasBase and implement get_projection_transform () routine?

Thanks.

**Comment by lcc815** [comment]
@TaoHuUMD Hi, is there any update?
Could you please give some example code here if you have finished this part of code?
Thanks a lot!!

**Reply by jacksonsunny29** [reply]
There's already one. You can check out pytorch3d.renderer.fisheyecamera
